classes/output/slide_table.php

```diff
@@ -88,47 +88,37 @@ public function populate_block_table($blockid) {
 
         $currorder = \block_carousel\local\slide_manager::get_current_order($blockid);
```

Comment:
get_current_order calls get_current_config which does a db call, can we move the caching up a layer or back a few lines to capture this?

Reply:
This data currently isnt cached. It would need to be rebuilt on every single order change, only to save a DB call on the edit menu. It isnt needed on block view, as the config is already loaded and dealt with by the block API. I dont currently think its worth it to save 1 DB call vs dev time to add caching and invalidation for this.